src/shared/components/query/QueryAndDownloadTabs.tsx

```diff
@@ -32,6 +32,7 @@ export default class QueryAndDownloadTabs extends React.Component<IQueryAndDownl
 	onSelectTab = (eventKey:string) =>
 	{
 		this.store.forDownloadTab = eventKey === DOWNLOAD;
+		this.store.selectableSelectedStudyIds = [];
```

Comment:
won't this undo selection when you move between tabs?  maybe that desirable?

Reply:
it shows `x studies selected (xxx samples)` in header if there are any virtual studies selected from query tab